cea/technologies/network_layout/main.py

```diff
@@ -82,16 +82,19 @@ class NetworkLayout(object):
 
     def __init__(self, network_layout=None):
         self.network_type = "DC"
-        self.connected_buildings = []
+        # self.connected_buildings = []
+        self.connected_buildings = network_layout.connected_buildings
         self.disconnected_buildings = []
         self.pipe_diameter = 150
```

Comment:
should this be called from the constants instead?

Reply:
This is not really part of the new optimisation script. But this would indeed be cleaner and doesn't change the functionality of the code, so I changed it.